awx/main/scheduler/task_manager.py

```diff
@@ -576,8 +576,9 @@ def schedule(self):
         with advisory_lock('task_manager_lock', wait=False) as acquired:
             with transaction.atomic():
                 if acquired is False:
-                    logger.debug("Not running scheduler, another task holds lock")
+                    logger.warning("Not running scheduler, another task holds lock")
                     return
                 logger.debug("Starting Scheduler")
                 with task_manager_bulk_reschedule():
                     self._schedule()
+                logger.debug("Finishing Scheduler")
```

Comment:
Yes please. You can _probably_ use the log timestamp between the start and finish debug message, but would be nice to explicitly print the time taken.